examples/helmet_detection/README.md

```diff
@@ -0,0 +1,192 @@
+# Using Incremental Learning Job in Helmet Detection Scenario
+
+This document introduces how to use incremental learning job in helmet detectioni scenario. Using the incremental learning job, our application can automatically retrains, evaluates, and updates models based on the data generated at the edge.
+
+## Helmet Detection Experiment
+
+### Install Neptune
+
+Follow the [Neptune installation document](/docs/setup/install.md) to install Neptune.
+
+### Prepare Data and Model
+
+Download dataset and model to your node:
+* step 1: download [dataset](https://edgeai-neptune.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/dataset.tar.gz)
+```
+mkdir -p /data/helmet_detection
+cd /data/helmet_detection
+tar -zxvf dataset.tar.gz
+```
+* step 2: download [base model](https://edgeai-neptune.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/model.tar.gz)
+```
+mkdir /model
+cd /model
+tar -zxvf model.tar.gz
+```
+### Prepare Script
+Download the [scripts](/examples/helmet_detection/training) to the path `code` of your node
+
+
+### Create Incremental Job
+
+Create Dataset
+
+```
+kubectl create -f - <<EOF
+apiVersion: neptune.io/v1alpha1
+kind: Dataset
+metadata:
+  name: incremental-dataset
+  namespace: neptune-test
```

Comment:
namespace `neptune-test` should be created first

Reply:
create neptune-test namespace first